warehouse/templates/manage/account.html

```diff
@@ -168,6 +168,8 @@ <h2 class="sub-title">Account details</h2>
     <div class="form-group">
       <span class="form-group__label">Username</span>
       <p class="form-group__text">{{ user.username }}</p>
+      <span class="form-group__label">Date Joined</span>
+      <p class="form-group__text"><i class="fa fa-calendar-alt" aria-hidden="true"></i> {{ user.date_joined|format_date() }}</p>
```

Comment:
`user.date_joined` might be `None` here, we need to guard against this similar to:

https://github.com/pypa/warehouse/blob/aaea30e44e6fe0b2ac6d948d91f965d03f98d885/warehouse/templates/accounts/profile.html#L31-L33

Reply:
Added the check in. I thought it was best to remove the icon since no other fields on the page had icons. Let me know what you think.

<img width="399" alt="Screen Shot 2019-05-06 at 3 53 34 PM" src="https://user-images.githubusercontent.com/15970628/57251219-44107a00-7017-11e9-9ab4-776aa54389ba.png">